server-common/src/main/java/org/apache/kafka/server/util/json/JsonValue.java

```diff
@@ -26,18 +26,16 @@
 /**
  * A simple wrapper over Jackson's JsonNode that enables type safe parsing via the `DecodeJson` type
  * class.
- *
+ * <br>
  * Typical usage would be something like:
- *
- * {{{
+ * <pre><code>
```

Comment:
It looks like we forgot to update the code example when we converted this class to Java, the snippet below is still in Scala.

Reply:
Oh good catch! I'll update this and the other feedback

Reply:
Done! Thanks for the feedback @mimaison !